luci-app-olsrd2/root/etc/uci-defaults/41_luci-olsrd2

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+uci -q batch <<-EOF >/dev/null
+	delete ucitrack.@olsrd2[-1]
+	add ucitrack olsrd2
+	set ucitrack.@olsrd2[-1].init=olsrd2
+	commit ucitrack
+EOF
+
+rm -f /tmp/luci-indexcache
+rm -f /tmp/luci-modulecache/*
```

Comment:
Why? This was already removed by treewide commit in LuCI and you are adding it back
- https://github.com/openwrt/luci/commit/1f8a3c04178e21f45f3908fd476ec3b3dad73c69